corehq/warehouse/etl.py

```diff
@@ -54,8 +62,9 @@ def _table_context(cls, start_datetime, end_datetime):
         for dep in cls.dependencies():
             dep_cls = get_cls_by_slug(dep)
             context[dep] = dep_cls._meta.db_table
-        context['start_datetime'] = start_datetime
-        context['end_datetime'] = start_datetime
+        context['start_datetime'] = start_datetime.isoformat()
```

Comment:
ok, no that's fine. wasn't sure if the T throws it off, but as long as it works i think it's fine